### ISSUE TITLE
Accessibility - Missing Country Flags (iOS and Android)

### ISSUE THREAD

**Issue by nmangu** [USDepartmentofLabor/Child-Labor#69]
Ref. Earlier issue #67 

The flags for the countries 'West Bank & Gaza Strip' (iOS & Android), and 'Western Sahara' (Android) are not visible. Screenshots attached.

![country flag missing_ios](https://cloud.githubusercontent.com/assets/10868918/18635060/9be90bee-7e51-11e6-8ca9-52a23216ca17.PNG)
![country flags missing_android](https://cloud.githubusercontent.com/assets/10868918/18635063/9c534658-7e51-11e6-97f1-78799a60340e.png)

**Comment by aquilent-dgemoets** [comment]
As you can see in the referenced #67, this is working as intended and the issue has been closed.

**Comment by nmangu** [comment]
@aquilent-dgemoets Thank you for attracting my attention to this. I probably skipped the line where it'd said 'there should be no flags for Western Sahara and West Bank & Gaza Strip.'. IT makes sense now. Thsi can be closed now.
